### ISSUE TITLE
Whitespace does not get shown with DiffPrettyText

### ISSUE THREAD

**Issue by crthpl** [sergi/go-diff#116]
it would be helpful if whitespace was shown. It could be shown with a Coloured background. Newlines could be shown as `\n` (although only if it is changed?)

**Comment by wormi4ok** [comment]
Agree. I was puzzled for some time when nothing was highlighted in the `DiffPrettyText` output, even though the diff was not empty.

**Comment by Broderick-Westrope** [comment]
This was a deal breaker for my project. Regularly diffing inconsistencies in whitespace was very annoying so I stopped using this tool. Other than that it was good. Would love to see better whitespace support for pretty printing.
